# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/fake/mythvideo/gcc34.patch
# ROCK Linux is Copyright (C) 1998 - 2005 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./mythvideo/videogallery.cpp.orig	2004-09-14 11:22:45.069889056 +0200
+++ ./mythvideo/videogallery.cpp	2004-09-14 11:23:01.582378776 +0200
@@ -17,6 +17,7 @@
 #include <qstringlist.h>
 #include <qpixmap.h>
 #include <unistd.h>
+#include <math.h>
 
 using namespace std;
 
